playwright.config.ts

```diff
@@ -13,7 +13,7 @@ if (process.env.CURRENTS_PROJECT_ID && process.env.CURRENTS_RECORD_KEY) {
 
 export default defineConfig({
   testDir: 'playwright',
-  fullyParallel: true,
+  fullyParallel: false,
   workers: 2,
```

Comment:
Not sure if this will do the trick? https://playwright.dev/docs/api/class-testconfig#test-config-fully-parallel
I think it would be safer to set the worker amount to 1 as well?
```suggestion
  workers: 1,
```